### ISSUE TITLE
Codecs for Native and JS types

### ISSUE THREAD

**Issue by aponos** [typelevel/skunk#1119]
Hi 👋!

I would like to contribute, adding some codecs for the Native and JS platforms. For example, the `bytea` type can also be converted to JS `UInt8Array`.

I just have one question regarding the code style of skunk. Those codecs should be included in the `core` project? Or it is better to have them in their own project, like the `circe` and `refined` codecs?

**Comment by mpilquist** [comment]
Yeah, core project makes sense to me. Feel free to open a draft PR to get some feedback before doing a whole bunch of codecs.

**Comment by aponos** [comment]
Thank you for your response @mpilquist!

I have opened the draft PR, exposing the idea of the `UInt8Array` to `bytea` codec. What do you think about the package `scalajs.codec` and the codec name?
